MachineLearningPrototype/utils/preprocess_data.py

```diff
@@ -15,3 +16,49 @@ def extract_quotation_content(content):
     if (len(substring) >= 1):
         result=' '.join(substring)
         return result
+
+def get_test_output(jenkins_url, test_name, config_file_path="./config.json"):
+    #Get servers to query
+    with open(config_file_path) as config_file:
+        config = json.loads(config_file.read())
+        if "servers" not in config:
+            raise Exception("No servers in config file")
+            return ""
+        servers = config["servers"]
+    
+    #Get build info from TRSS
+    query_url = "https://trss.adoptium.net/api/parseJenkinsUrl"
+    query_params = {"jenkinsUrl": jenkins_url}
+    output = requests.get(query_url, query_params)
+    output_dict = json.loads(output.content)
+
+    if "output" not in output_dict:
+        raise Exception("Invalid jenkins Url")
+        return ""
+    
+    output_dict = output_dict["output"]
+    success = (output_dict["errorMsg"] == "")
+    
+    if not success:
```

Comment:
`if not output_dict["errorMsg"]` will simplify the code here.

Reply:
Done! Thank you.